Makefile

```diff
@@ -56,7 +56,7 @@ integration:
 	go test -v sigs.k8s.io/cluster-api-provider-aws/test/integration
 
 fmt:
-	hack/verify-gofmt.sh
+	hack/go-fmt.sh .
```

Comment:
Is go-fmt different to go-vet w.r.t. passing `./...`. One does, the other doesn't. Curious as to why.

Reply:
No idea, that's how installer runs it.
All those scripts are created by guys from installer team, we just copy-pasted them.

Reply:
You just need to look at the code of the script https://github.com/openshift/cluster-api-provider-aws/blob/master/hack/go-fmt.sh#L4 https://golang.org/cmd/gofmt/

Reply:
You just need to look at the code of the script https://github.com/openshift/cluster-api-provider-aws/blob/master/hack/go-fmt.sh#L4 https://golang.org/cmd/gofmt/